forch/forchestrator.py

```diff
@@ -100,14 +102,25 @@ def initialize(self):
         """Initialize forchestrator instance"""
         self._metrics = ForchMetrics(self._config.varz_interface)
         self._metrics.start()
-        self._faucet_collector = FaucetStateCollector(self._config.event_client)
+        self._faucet_collector = FaucetStateCollector(self._config)
         self._faucet_collector.set_placement_callback(self._process_device_placement)
         self._faucet_collector.set_get_dva_state(
             (lambda switch, port:
              self._faucetizer.get_dva_state(switch, port) if self._faucetizer else None))
         self._faucet_collector.set_forch_metrics(self._metrics)
         self._faucet_state_scheduler = HeartbeatScheduler(interval_sec=1)
-        self._faucet_state_scheduler.add_callback(self._faucet_collector.heartbeat_update)
+        self._faucet_state_scheduler.add_callback(self._faucet_collector.heartbeat_update_stack_state)
+
+        gauge_metrics_interval_sec = 10
```

Comment:
10 should be configurable w/ default  (so should the interval_sec above)

Reply:
Made it configurable. Forch will check the packet rate only when the interval is configured.